Makefile

```diff
@@ -97,7 +97,7 @@ goinstall:
 	${INSTALL_CMD} ${BASE}/cmd/hostagent
 	${INSTALL_CMD} ${BASE}/cmd/acikubectl
 
-dist/opflex-agent-cni: ${AGENTCNI_DEPS} 
+dist/opflex-agent-cni: ${AGENTCNI_DEPS}
```

Comment:
Not related to this PR. But, we should get rid of the deps business I think. It is not consistent with go. We should use deps only for the secondary artifacts generated from go binaries.